src/react/Layout.jsx

```diff
@@ -97,19 +103,9 @@ const Layout = props => {
 
 Layout.propTypes = {
 	documentTitle: PropTypes.func.isRequired,
-	deactivateError: PropTypes.func.isRequired,
```

Comment:
This change is required because of the Not Found route which results in the below console warning:

https://github.com/andygout/dramatis-cms/blob/1e65b09ea44c07257e33f5f727db7ac666c9bede/src/react/routes.js#L273

> Warning: Failed prop type: The prop `deactivateError` is marked as required in `Layout`, but its value is `null`.

This warning was appearing upon visits to the Not Found route so this change was required to fix that in any event, but with the changes made by this PR, the warning appears for any route that is visited, and so has been included in this PR for that reason.